docs/videos/running_all_devs.md

```diff
@@ -0,0 +1,41 @@
+# Running the All Core Devs call
+
+Create draft agenda file titled “alldevs[date].md” with the date of the next call in YYYYMMDD format.
+- Update the mkdocs.yml file of the wiki with a relative link to the file under the Videos > Meetings > All Core Devs > Agendas section.   
+- Add link to the agenda to the [Overview](dev.md) page in the wiki.
+
+Review proposed agenda items
+ - Check alldevs email address throughout the week for proposed agenda items. Review proposed items and respond with [accepted](all_devs_accepted.md) or [rejected](all_devs_rejected.md) templates.
+ - Add approved agenda items and the email addresses of agenda item owners to this meeting’s “Agenda + Guest Participants for [date]” email, replacing [date] with the date of the meeting.
+  - All approved agenda items to the next call’s agenda in the wiki.
+
+Ten mins before scheduled to begin. Create [Jitsi](https://meet.jit.si) channel with unique link.
```

Comment:
I think it'd be easier for note taking / spontaneous agenda items during the call if we copy the agenda into a google doc and then take notes in that google doc.

Afterwards, we can transfer the contents of the google doc back into the wiki, along with assignees of any actions items.

Reply:
Add info about this.